Tasks/Maven/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -23,5 +24,17 @@
   "loc.input.label.jdkUserInputPath": "JDK Path",
   "loc.input.help.jdkUserInputPath": "Sets JAVA_HOME to the given path.",
   "loc.input.label.jdkArchitecture": "JDK Architecture",
-  "loc.input.help.jdkArchitecture": "Optionally supply the architecture (x86, x64) of JDK."
+  "loc.input.help.jdkArchitecture": "Optionally supply the architecture (x86, x64) of JDK.",
+  "loc.input.label.sqAnalysisEnabled": "Run SonarQube Analysis",
+  "loc.input.help.sqAnalysisEnabled": "Run a [SonarQube](http://www.sonarqube.org/) analysis after executing the current goals. 'install' or 'package' goals should be executed first.",
+  "loc.input.label.sqConnectedServiceName": "SonarQube Endpoint",
+  "loc.input.help.sqConnectedServiceName": "The SonarQube server generic endpoint",
+  "loc.input.label.sqDbDetailsRequired": "The SonarQube Server version is lower that 5.2",
+  "loc.input.help.sqDbDetailsRequired": "If using a SonarQube server 5.1 or lower you need to specify the database connection details.",
+  "loc.input.label.sqDbUrl": "Db Connection String",
+  "loc.input.help.sqDbUrl": "SonarQube server 5.1 and lower only. The database connection setting, i.e. sonar.jdbc.url. For example jdbc:jtds:sqlserver://localhost/sonar;SelectMethod=Cursor",
+  "loc.input.label.sqDbUsername": "Db UserName",
+  "loc.input.help.sqDbUsername": "SonarQube server 5.1 and lower only The username for the database user, i.e. sonar.jdbc.username.",
```

Comment:
Missing full stop after "only". There shouldn't be a comma before the "i.e." (same in lines 35 and 39).